Project/Wikode/wikodeApp/views.py

```diff
@@ -88,7 +89,8 @@ def homePage(request):
 def articleDetail(request, pk):
     article = Article.objects.get(pk=pk)
     wiki_info = {}
-
+    activity_manager = ActivityManager(user_id=request.user.id)
```

Comment:
request tipine göre iflerin altına yazılmalı. Sadece "GET" requestler bu view için view activitydir.
"POST" requestler tagging activilere karşılık geliyor.